hyperspy/exceptions.py

```diff
@@ -190,6 +190,15 @@ def __init__(self, navigation_size, expected_navigation_size):
             self.navigation_size, self.expected_navigation_size)
 
 
+class NonLinearAxisError(Exception):
+
+    def __init__(self):
+        self.error = ("Non linear axis are not supported.")
```

Comment:
Should be either "axis is" or "axes are"

Reply:
OK